common/cah-model.cpp

```diff
@@ -73,16 +73,16 @@ bool cah_model::prompt_trigger_camera_accuracy_health(device_model & dev_model,
                     sd->s->set_option(RS2_OPTION_TRIGGER_CAMERA_ACCURACY_HEALTH, 1.0f);
                     device_calibration dev_cal(dev_model.dev);
                     // Register AC status change callback
-                    if (!registered_to_callback)
+                    if (!_registered_to_callback)
                     {
-                        registered_to_callback = true;
-                        cah_state = model_state_type::PROCESS_MODAL;
```

Comment:
This only happens the first time in here?

Reply:
Yes, if the user clicked yes it will register it to the callback function and set the state machine to the next state.

Reply:
I don't understand -- if it's not the first time, then we don't need to move to PROCESS_MODAL?

Reply:
fixed as agreed, register will happen once in the SW runtime life.